engine/schema/migrations/20170804152152_align_subtype_columns.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AlignSubtypeColumns extends AbstractMigration {
+	/**
+	 * Change Method.
+	 *
+	 * Write your reversible migrations using this method.
+	 *
+	 * More information on writing migrations is available here:
+	 * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+	 *
+	 * The following commands can be used in this method and Phinx will
+	 * automatically reverse them when rolling back:
+	 *
+	 *    createTable
+	 *    renameTable
+	 *    addColumn
+	 *    renameColumn
+	 *    addIndex
+	 *    addForeignKey
+	 *
+	 * Remember to call "create()" or "update()" and NOT "save()" when working
+	 * with the Table class.
+	 */
+	public function change() {
+
+		$options = [
+			'null' => false,
+			'limit' => 252,
+			'encoding' => "utf8",
+			'collation' => "utf8_general_ci",
+		];
+
+		$table = $this->table('river');
+		$table->changeColumn('subtype', 'string', $options)->save();
```

Comment:
As seen in the docblock above, the `change()` method does not support `changeColumn()`, as it's unable to track back the changes for migrating down. I would suggest using `up()` and `down()` instead in case `change()` does not support it to fully support rolling back.